src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs

```diff
@@ -54,6 +54,30 @@ public bool UseCookies
             }
         }
 
+        public TimeSpan? FakeRtt
```

Comment:
I assume these properties are just for testing purposes?

Do we think we would want to expose any of these publicly?

Reply:
No, all these extra properties are for testing/benchmarking only.